ajax/libs/query-string/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "query-string",
+  "filename": "index.min.js",
+  "description": "Parse and stringify URL query strings",
+  "keywords": [
+    "browser",
+    "querystring",
+    "query",
+    "string",
+    "qs",
+    "param",
+    "parameter",
+    "url",
+    "uri",
+    "parse",
+    "stringify",
+    "encode",
+    "decode"
+  ],
+  "author": "Sindre Sorhus <sindresorhus@gmail.com> (sindresorhus.com)",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SaneMethod/sindresorhus/query-string.git"
+  },
+  "npmName": "query-string",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "index.js"
```

```suggestion
        "index.*"
```
We should also include the typescript typings for those that may want it.